apps/etterlatte-behandling/src/main/kotlin/sak/SakDao.kt

```diff
@@ -107,4 +83,11 @@ class SakDao(private val connection: () -> Connection) {
             } > 0
         }
     }
+
+    private fun ResultSet.toSak() = Sak(
+        sakType = enumValueOf(getString(2)),
```

Comment:
Usikker på om jeg for dette, er så lett å få feil svar siden man bruker denne pga indeks uthentingen.

Reply:
Einig. Var ikkje så mykje betre før heller, så vi kunne nok med fordel ha skrive om heile denne klassa til Kotliquery. Kan sjå på det (kanskje i ein eigen PR, eventuelt inn mot denne) viss eg får ånden over meg til å jobbe inn nokre timar.

Reply:
Kikka litt på det, og lurer på om vi bør finne ut av kor/korleis vi vil ha transaksjonsstyringa før denne omskrivinga kjens heilt bra/trygg.

Reply:
Ref kotliquery - god plan - men egen PR kanskje?